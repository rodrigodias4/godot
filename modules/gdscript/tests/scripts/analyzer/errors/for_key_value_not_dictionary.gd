func test():
    for u, v in ["test1", "test2", "test3"]:
        print(u, " ", v)
