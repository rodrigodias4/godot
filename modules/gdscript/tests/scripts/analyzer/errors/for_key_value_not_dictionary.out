GDTEST_ANALYZER_ERROR
Unable to iterate with key-value pair on iteration of type "Array" (not a dictionary).
